### ISSUE TITLE
Adding addresses to personal address book failes

### ISSUE THREAD

**Issue by derherrfreitag** [stwa/google_addressbook#53]
Using Roundcube 1.4.2, saving address to personal address book fails with:

```
PHP Fatal error:  Uncaught Error: Call to undefined function write_log() in [...]/roundcubemail/plugins/google_addressbook/google_addressbook.php:195
Stack trace:
#0 [...]/roundcubemail/program/lib/Roundcube/rcube_plugin_api.php(441): google_addressbook->contact_create(Array)
#1 [...]/roundcubemail/program/steps/addressbook/save.inc(220): rcube_plugin_api->exec_hook('contact_create', Array)
#2 [...]/roundcubemail/index.php(302): include_once('/var/www-sites/...')
#3 {main}
    thrown in [...]/roundcubemail/plugins/google_addressbook/google_addressbook.php on line 195
```

Changing write_log(...) to rcube::write_log(...) (on all occurences) fixes this.

**Comment by derherrfreitag** [comment]
Important note: The proposed fix will lead to address data in the logs, because it dumps the address struct. If you do not want this (I would assume this is the case for almost all setups, that are not for private use), comment out the three occurences.

**Comment by derherrfreitag** [comment]
This is a duplicate to issue #48, however above comment holds true for the fix of #48 as well
